data/example-apps.yaml

```diff
@@ -0,0 +1,11 @@
+title: Example Apps
```

Comment:
I don't we need this is a separate category anymore, should just be folded into apps or tools

Reply:
Yeah, yeah, I've changed that right after pushing. I didn't push the new commits yet :p Thanks for noticing though.